k8s/manifests/kube-prometheus/lib/simple-server/database-dashboard.libsonnet

```diff
@@ -84,7 +84,7 @@ local database =
     ),
   ]);
 
-g.dashboard.new('Simple Database Dasboard')
+g.dashboard.new('Simple Database Dashboard')
```

Comment:
Here and elsewhere, I've sneaked in some random typo fixes that were bothering me.